build/build-proto.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -o errexit
+set -o xtrace
+
+if ! swagger version &>/dev/null; then
+  go get github.com/go-swagger/go-swagger/cmd/swagger@v0.24.0
+fi
+
+rm -rf versionserviceclient
+
+swagger generate client -f vendor/github.com/Percona-Lab/percona-version-service/api/version.swagger.yaml -c versionserviceclient -m versionserviceclient/models
```

Comment:
File name does not look right

Reply:
it works correctly

Reply:
Ok, but the name of the file is `build-proto`. But there is no any build proto commands

Reply:
@blez 
Ah, you mean script name.I thought the name of `version...` from vendor folder

Reply:
44ddf3e065b3ecc06b9370cb2635635d2048dfed